ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/product/v1/product/README.md

```diff
@@ -31,7 +31,7 @@ The Product component uses the `com.adobe.cq.commerce.core.components.models.pro
 
 ### Selectors & Request Parameters
 This component is targeted for a product page showing details of a single simple or configurable product.
-1. The product SKU is retrieved form the first URL selector. 
+1. The product slug is retrieved form the first URL selector. Alternatively, the product sku can be defined via the edit dialog.
```

Comment:
Actually the selector might either be the `slug` or `sku` depending how the `UrlProvider` is configured, the documentation was outdated here. Can you please update this?